### DISCUSSION TITLE
Alarm plays non-stop when you are using some APP playing sound like whatsapp call

### DISCUSSION THREAD

**Discussion by rodrigowebjump** [Navid200/xDrip#110] [Alerts]
When I am using a whatsapp call and the alarm plays the sound, it does not stop and plays indefinitely until I restart the mobile phone.

**Comment by Navid200** [comment]
Thanks for reporting this.

Which alarm?
An alarm on the Glucose Level Alerts List? 
Or one of predicted low or persistent high alerts?

So, if you snooze the alert, it ignores it and keeps going?

**Reply by starbuck93** [reply]
I've also experienced this and the only fix I found is rebooting, as well. I think mine was a "BG Rising" alarm. But I also think it only did it on bluetooth audio. Snoozing all alerts didn't stop the alarm.

**Comment by Navid200** [comment]
We have discussions now opened in the main repository:
https://github.com/NightscoutFoundation/xDrip/discussions

I am looking for a way to transfer the open discussions from here to there. But, I'm not sure I'll succeed. In that case, it may be easier to open new discussions there.
